spi/policy-spi/src/main/java/org/eclipse/dataspaceconnector/spi/policy/store/PolicyStore.java

```diff
@@ -15,6 +15,7 @@
 package org.eclipse.dataspaceconnector.spi.policy.store;
```

Comment:
Shouldn't this be called `PolicyDefinitionStore` for coherence?

Reply:
yup, you're right. will refactor.